_layouts/default.html

```diff
@@ -19,7 +19,7 @@
             <hr class="pldoc-divider sr-only">
 
             <div class="pldoc-wrapper-content-main">
-                <main class="pldoc-content pldoc-content-main" id="content-main">
+                <main class="pldoc-content pldoc-content-main" id="content-main" aria-label="Main">
```

Comment:
Is this needed? I thought the ``<main>`` element already gave assistive tech a ton of Main-ish things by default, including a hint that this was the main landmark of a view.